### ISSUE TITLE
Exception Handling & Accessibility

### ISSUE THREAD

**Issue by rtk-rnjn** [rtk-rnjn/MomCare#280]
1. Respect Reduce Motion, Voice Over and other accessibility traits, hints, labels, etc wherever missing.
2. do catch all try? and alert on error
